packages/katello/katello/katello-remove

```diff
@@ -43,7 +44,6 @@ LOG_FILES=[
   '/var/log/pulp',
   '/var/log/candlepin',
   '/var/log/httpd',
-  '/var/log/mongodb',
```

Comment:
I think these should still be cleaned. We also have `/var/log/tomcat6` in here which only exists on EL6. We should also clean out the logs from the new location.